pkg/tanka/load.go

```diff
@@ -36,6 +36,9 @@ func LoadEnvironment(path string, opts Opts) (*v1alpha1.Environment, error) {
 		return nil, err
 	}
 
+	if opts.Name != "" {
+		opts.JsonnetOpts.EvalScript = fmt.Sprintf(SingleEnvEvalScript, opts.Name)
+	}
```

Comment:
Performance nit: This is only relevant for `inline` environments, and a performance cost for `static` environments, as the Jsonnet VM needs to iterate over the entire Jsonnet object tree searching for the `tanka.dev/Environment` it will never find. This can add multiple seconds to the evaluation time.

We should optimize this (in a future iteration?), e.g. by moving this code into `InlineLoader` (loaders need their own Opts type then)

Reply:
I would like to hear suggestions on that, different Ops could be a viable solution.